test/integration/src/html.pre.js

```diff
@@ -21,3 +22,89 @@ module.exports.pre = (payload, action) => {
   payload.content.stamp = utils.stamp() + utils2.stamp() + utils3.stamp();
   payload.resourcePath = action.request.params.path;
 };
+
+module.exports.before = {
+  fetch: (payload) => {
+    if (payload.request.path==='/404.html') {
+      return {
+        content: {
+          body: 'This file does not exist.'
+        }
+      }
+    }
+  }
+}
+
+/**
+ * Finds MDAST objects that look like this:
+ * ```markdown
+ * > [!NOTE]
+ * > Content of the note
+ * ```
+ * and flags them.
+ * @param {object} node an MDAST node
+ */
+function findnotes(node) {
+  if (node && 
+    node.type === 'blockquote' && 
+    node.children && 
+    node.children[0] && 
+    node.children[0].children && 
+    node.children[0].children[0] && 
+    node.children[0].children[0].type === 'linkReference' &&
+    node.children[0].children[0].identifier && 
+    node.children[0].children[0].identifier.match(/![a-z]+/)) {
+    // get the class name
+    const classname = node.children[0].children[0].identifier.replace(/!/, 'admonition-');
+    const title = node.children[0].children[0].identifier.replace(/!/, '');
+
```

Comment:
`node.children[0].children[0]` are scary (but only way to do it for now I think). In the future, we might need to add helper method on the mdast nodes to avoid this.

Reply:
I think with https://github.com/syntax-tree/unist-util-select I could have written it a bit more elegantly.

Reply:
At least that's what I'd have used if I had to write the same functionality for `helix-pipeline`

Reply:
or use: https://lodash.com/docs/4.17.11#at